.github/workflows/backport.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     name: Backport
     steps:
       - name: Backport
-        uses: tibdex/backport@v1
+        uses: VachaShah/backport@v1.1.4
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch_name: backport/backport-${{ github.event.number }}
```

Comment:
Is this going to create stale branches every time a PR is updated that has a backport label?

Reply:
A branch will only be created once the PR is merged. So if a PR number 100 is to be backported to release branch 1.x, the branch that will be created in the process is `backport/backport-100-to-1.x` after the PR is merged.